runtime/jcl/exports.cmake

```diff
@@ -435,6 +434,12 @@ omr_add_exports(jclse
 	Java_sun_reflect_ConstantPool_getUTF8At0
 )
 
+if(JAVA_SPEC_VERSION LESS 24)
+omr_add_exports(jclse
+	Java_java_security_AccessController_initializeInternal
+)
+endif()
```

Comment:
Please add this near the end, keeping the conditional exports organized by version (after the Java 20+ block).